gallery/src/components/NxToast/NxToastComplexLayoutExample.tsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { useState} from 'react';
+import { reject, propEq } from 'ramda';
+
+import {
+  NxGlobalSidebarNavigation,
+  NxGlobalSidebarNavigationLink,
+  NxButton,
+  NxButtonBar,
+  NxBackButton,
+  NxStatefulGlobalSidebar,
+  NxPageMain,
+  NxPageTitle,
+  NxH1,
+  NxToastContainer,
+  NxToast,
+  NxSuccessAlert,
+  NxWarningAlert,
+  NxErrorAlert,
+  NxInfoAlert,
+  NxTile,
+  NxP
+} from '@sonatype/react-shared-components';
+import {
+  faArrowLeft,
+  faArrowRight,
+  faLink
+} from '@fortawesome/free-solid-svg-icons';
+
+const sidebarLogoPath = require('../../assets/images/logo-plaid-villain-text.png');
+
+type ToastTypes = 'success' | 'error' | 'info' | 'warning';
```

Comment:
Go ahead and update this example too to get rid of this complexity

Reply:
> Go ahead and update this example too to get rid of this complexity

updated here: [c170e87](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/c170e87da1c5749296221fa0c870bd5fab2de6da)